back/node_watcher/docker-compose.yml

```diff
@@ -1,15 +1,23 @@
-version: '3'
+version: '3.7'
 services:
-  # node_watcher:
-  #   image: kusama_node_watcher
-  #   restart: always
-  #   ports:
-  #     - "3000:3000"
+  main:
```

Comment:
renamed to main, or else `node_watcher_node_watcher_1` was displayed, instead of now `node_watcher_main_1`